index.js

```diff
@@ -138,7 +139,9 @@ module.exports = {
     "require-yield": 1,
     "space-infix-ops": 1,
     "space-in-parens": [1, "never"],
-
+    "space-before-blocks": 1,
+    "func-style": [1, "declaration", { "allowArrowFunctions": true }],
```

Comment:
That's OK, that's what the `allowArrowFunctions` part does :-)

<img width="928" alt="screen shot 2017-03-08 at 1 05 14 pm" src="https://cloud.githubusercontent.com/assets/762070/23703382/f0e47096-03ff-11e7-9fbd-4e44e2972e2e.png">

Line 12 needs to be rewritten but line 11 can stay that way.